cg/services/order_validation_service/rules/sample/utils.py

```diff
@@ -85,3 +85,10 @@ def is_invalid_well_format(sample: Sample) -> bool:
     if sample.is_on_plate:
         return not bool(re.match(correct_well_position_pattern, sample.well_position))
     return False
+
+
+def is_invalid_container_name(sample: Sample) -> bool:
+    """Checks if a sample in a plate has an invalid container name."""
```

Comment:
In general I do not think methods like this needs a docstring, but that is probably just flavour. I would say that this phrasing makes it sound as if it can only be used for samples on plates though